src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -363,7 +374,7 @@ public function editRootAction($categoryId, Request $request)
         $rootCategoryFormHandler = $this->get('prestashop.core.form.identifiable_object.handler.root_category_form_handler');
 
         try {
-            $rootCategoryForm = $rootCategoryFormBuilder->getFormFor((int) $categoryId);
+            $rootCategoryForm = $rootCategoryFormBuilder->getFormFor((int) $categoryId, []);
```

Comment:
```suggestion
            $rootCategoryForm = $rootCategoryFormBuilder->getFormFor((int) $categoryId);
```
If you don't need to override the data then you don't need this second parameter (default value is already `[]`)